extensions/amp-story-audio-sticker/0.1/amp-story-audio-sticker.css

```diff
@@ -0,0 +1,18 @@
+amp-story-audio-sticker {
+  z-index: 3 !important;
+  position: absolute !important;
+  height: auto !important;
```

Comment:
What are these for? Are we positioning the sticker ourselves?

Reply:
Removed.